src/components/FormInput/FormInput.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import { DEFAULT_PROPS } from './FormInput.constants';
+import { StyledFormInput } from './FormInput.styles';
+
+import type { FormInputProps } from './FormInput.types';
+
+export const FormInput: React.FC<FormInputProps> = ({
+  variant = DEFAULT_PROPS.variant,
+  type = DEFAULT_PROPS.type,
+  size = DEFAULT_PROPS.size,
+  placeholder,
+  value,
+  onChange,
+  errorMessage,
+  disabled = DEFAULT_PROPS.disabled,
+  onFocus,
+  onBlur,
+  'data-testid': dataTestId,
+  ...rest
+}) => {
+  const isError = !!errorMessage;
+
+  const handleFocus = (event: React.FocusEvent<HTMLInputElement>) => {
+    if (onFocus) onFocus(event);
+  };
+
+  const handleBlur = (event: React.FocusEvent<HTMLInputElement>) => {
+    if (onBlur) onBlur(event);
+  };
+
+  return (
+    <StyledFormInput
+      variant={variant}
+      size={size}
+      type={type}
+      placeholder={placeholder}
+      value={value}
+      onChange={onChange}
+      error={isError}
+      helperText={isError ? errorMessage : undefined}
+      disabled={disabled}
+      onFocus={handleFocus}
+      onBlur={handleBlur}
+      data-testid={dataTestId}
+      {...rest}
+      inputProps={{ 'aria-label': placeholder || 'input' }}
+    />
+  );
+};
+
+export default FormInput;
```

Comment:
This file exports the component as a default export (export default FormInput). The project rules state that named exports must be used and default exports are forbidden.

**Sugerencia:**
```typescript
Remove the default export and only use the named export already present. For example, delete line 52. Consumers should import with: import { FormInput } from './FormInput';
```